packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue

```diff
@@ -22,18 +22,38 @@ export default {
     Simplebar,
     SfButton,
   },
+  inject: {
+    contentShow: {
+      default: "contentShow",
+    },
+  },
   props: {
     maxContentHeight: {
       type: String,
-      default: "",
+      default() {
```

Comment:
![Zrzut ekranu 2020-05-8 o 13 17 00](https://user-images.githubusercontent.com/12138170/81401718-32811700-9130-11ea-9584-3c7eff854be8.png)
Like you see, you can't use data from anywhere here. You can use just static data.

Reply:
I removed that, but I found this in Vue docs: 
`The next 2 examples work with Vue 2.2.1+. Below that version, injected values were resolved after the props and the data initialization.

Using an injected value as the default for a prop:

const Child = {
  inject: ['foo'],
  props: {
    bar: {
      default () {
        return this.foo
      }
    }
  }
}`
So I thought that I can you it :)

Reply:
oh good to know, thank you! :)